### PR TITLE
88 fix nonetype errors from dataclasses json

### PR DESCRIPTION
**Pull request by evalott100**

### CODE REVIEW

File: src/artemis/devices/eiger.py [hunk 1]
```diff
@@ -159,7 +158,7 @@ def arm_detector(self):
         odin_status &= await_value(self.odin.meta.ready, 1)
         odin_status.wait(10)
 
-        set_and_wait(self.cam.acquire, 1, timeout=10)
+        self.cam.acquire.set(1).wait(timeout=10)
```
Comment (DominicOram) [inline src/artemis/devices/eiger.py:161]: Good catch!

**Review by DominicOram — CHANGES_REQUESTED** [review]
Thanks to looking into this, it does remove the warnings. However, couple of bits:
* I don't think we should be setting these params to optional, in production they are always required to be set and having them optional implies that's not the case
* You have some code in here for the issue around swapping `image_size`, can you remove it from here and make sure it's just on the correct branch

**Comment by evalott100** [comment]
> I don't think we should be setting these params to optional, in production they are always required to be set and having them optional implies that's not the case

I agree, I think it was a quirk of the typing. I'll rewrite it to make it non-optional.

> You have some code in here for the issue around swapping image_size, can you remove it from here and make sure it's just on the correct branch

My bad, I'll start double checking this

**Review by DominicOram — APPROVED** [review]
Thanks, this reduces the warning significantly.

**Pull request merged**